src/components/ChallengeEditor/index.js

```diff
@@ -131,6 +134,29 @@ class ChallengeEditor extends Component {
     this.resetChallengeData(this.setState.bind(this))
   }
 
+  deleteModalLaunch () {
+    if (!this.state.isDeleteLaunch) {
+      this.setState({ isDeleteLaunch: true })
+    }
+  }
+
+  async onDeleteChallenge () {
+    const { deleteChallenge, challengeDetails, history } = this.props
+    console.log(challengeDetails, deleteChallenge)
```

Comment:
@mark-nakachon let's remove `console.log` as I guess it's only needed for debugging.